### ISSUE TITLE
Purpose of fork?

### ISSUE THREAD

**Issue by andrewbarbour** [joshirio/passiflora#2]
I have been following symphytum for a few years (business guy not a developer) and have tinkered with it -  hoping someone would pick up and run with the VERY cool foundation of the code base.  I see lots of potential applications....  what is different with the passiflora fork?  Is there a way to connect developer(s) to discuss vision and potential applications/opportunities?

**Comment by joshirio** [comment]
Passiflora is a fork of symphytum to build a specific database application intended for a small group of users that work with specific plants and needed a database to organize work.

It has all editing features removed and serves only as an interface to browse a database file which is downloaded at runtime from the internet from a specific website. This database file is not included in the code base and is release under a different license (with copyright) and requires permission to use.

While passiflora database software is just the open source viewing container for it.

More info at http://passiflora.enmed.de (German only though).

While working on passiflora I improved Symphytum at the same time, since it shares most of the code base. I became also a contributor to the Symphytum Project this way.

Passiflora and Symphytum are completely separate projects now and with different goals.
As time permits, I'm looking to contribute to both projects, since the database file deployed in passiflora is really created with Symphytum.

**Comment by andrewbarbour** [comment]
Ahh... makes sense.  Unsure if someone has developed something similar -
but I am trying to figure out how I would take the form design from the
application and then create a webpage that would point to the same DB in
dropbox in a read only mode....
AB

**Comment by joshirio** [comment]
Well, to do that it would require a web version of symphytum built in a web compatible framework like PHP 
 for backend and javascript/HTML for user interface, basically a complete rewrite because the used Qt toolkit is for desktop and mobile apps only.

**Comment by andrewbarbour** [comment]
I was thinking a script to dynamically publish the database (of course not
to large) to a temporary XML blob like thing and then have a standard page
that would point to the XML blob which would display a data grid and a
simple page for individual record viewing when a row was selected (detail
below table).

I have had someone build something similar (minus the dynamic publishing)...

Andrew

**Comment by joshirio** [comment]
Yes a static XHTML site as an export format from symphytum would be possible, much easier than rewriting the entire app in PHP/JS/HTML

**Comment by andrewbarbour** [comment]
I have the code which will display the data grid and the single record
below (including pagination) -  I will just have to figure out the dynamic
translation bit.

I can't believe that Symphytum has not exploded - the tool has so many
potential uses.  Personally - I think a rebranding may help its potential
adoption.
